json struct.py

```diff
@@ -88,24 +88,48 @@ def get_read_value(root, key, value, chain):
     chain = chain if len(chain) == 0 else f"{chain}."
     t = get_type(value)
 
+    ret_val = f"""
```

Comment:
I see what you've done here. It's messier code but better.
By messier I mean it doesn't belong in the python. 
I'd like to somehow move this to an external file so people looking to modify will not have to touch the python file and it would be easier to do.
A project for another time. For now it looks good.